### PR TITLE
Add the "log" command to admin CLI

### PR DESCRIPTION
**Pull request by dominikl**
Adds the "log"command also to the admin CLI plugin.
Can use that as a brief example for the CLI training session next week to show how to extend a plugin.

### CODE REVIEW

**Comment by joshmoore** [comment]
Nice. In terms of upgrades or providing feedback on the lists, this could be quite useful. It _does_ make me worry about the `repo` argument which isn't well though through (on my part). It could have been named "context" or similar. It's also quite like the glaring first argument to `SharedResources.newTable( repoId, ...)`. As long as we're happy having it optional (as with `newTable`), then likely there's nothing too large to modify here or in @mtbc's PR. Happy to discuss though if anyone sees the need.

**Comment by dominikl** [comment]
Is the `repoUuid` argument really optional @mtbc ? I thought I tried with `None` and it failed, but let me try again, not 100% sure. To be honest, I have no idea and what `repoUuid` actually does.

**Comment by mtbc** [comment]
I am not aware that it's optional, at least for my https://github.com/openmicroscopy/openmicroscopy/pull/5538 test script in the description I found it helped to include an irrelevant one. I think it somehow sets a repository context for the RawAccessRequest's operation.

**Comment by dominikl** [comment]
Shall I just remove the parameter from the admin CLI and pass on `ScriptRepo`? It doesn't seem to make a difference anyway, it just can't be `null`, respectively `None`.

**Comment by mtbc** [comment]
Maybe pick the name of a random one from http://downloads.openmicroscopy.org/omero/5.4.0/api/slice2html/omero/grid/SharedResources.html#repositories.

**Comment by joshmoore** [comment]
Yeah, it definitely has to be a valid UUID: https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/blitz/src/ome/services/blitz/repo/RawAccessRequestI.java#L100

So the only other options I can think of are:

 * define some global names that short circuit the lookup: "global" or "log"
 * move this out of RawAccess to a new cmd

**Comment by mtbc** [comment]
I suspect we're fine as-is: it's an admin-only debug utility that few have sought and it's easy to get a valid repo name.

If not: I'd originally thought maybe just an IAdmin method because it seems odd to wheel in a whole new async class for a single operation that is both fast and trivial. Still, if the implementation does need moving, I'd certainly appreciate help knowing to where, especially if good naming now might later help us to group related operations with it. (Or we could try to figure a good global repo name but other parts of RAR may then have to cope.)

**Comment by joshmoore** [comment]
@mtbc: can you go ahead and pull this into the mainline PR, then we can adapt the target branch and continue this conversation, potentially merging and re-iterating on the changes later.
